src/components/calcite-sortable-list/calcite-sortable-list.tsx

```diff
@@ -0,0 +1,169 @@
+
+  /**
+   * When true, disabled prevents interaction. This state shows items with lower opacity/grayed.
+   */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**
+   * When true, content is waiting to be loaded. This state shows a busy indicator.
+   */
+  @Prop({ reflect: true }) loading = false;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteSortableListElement;
+
+  @State() handleActivated = false;
+
+  items: Element[] = [];
+
+  observer = new MutationObserver(() => {
+    this.cleanUpDragAndDrop();
+    this.items = Array.from(this.el.children);
+    this.setUpDragAndDrop();
+  });
+
+  sortable: Sortable;
+
+  handle: Node = null;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  // --------------------------------------------------------------------------
+
+  componentDidLoad() {
+    this.items = Array.from(this.el.children);
+    this.setUpDragAndDrop();
+    this.observer.observe(this.el, { childList: true, subtree: true });
+  }
+
+  componentDidUnload() {
+    this.observer.disconnect();
+    this.cleanUpDragAndDrop();
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Emmitted when the order of the list has changed.
+   * @event calciteListOrderChange
+   */
+  @Event() calciteListOrderChange: EventEmitter;
+
+  @Listen("calciteHandleNudge") calciteHandleNudgeHandler(event: CustomEvent) {
+    const sortItem = this.items.find((item) => {
+      return item.contains(event.detail.handle) || event.composedPath().includes(item);
+    });
+
+    const lastIndex = this.items.length - 1;
+    const startingIndex = this.items.indexOf(sortItem);
+    let appendInstead = false;
+    let buddyIndex;
+    switch (event.detail.direction) {
+      case "up":
+        event.preventDefault();
+        if (startingIndex === 0) {
+          appendInstead = true;
+        } else {
+          buddyIndex = startingIndex - 1;
+        }
+        break;
+      case "down":
+        event.preventDefault();
+        if (startingIndex === lastIndex) {
+          buddyIndex = 0;
+        } else if (startingIndex === lastIndex - 1) {
+          appendInstead = true;
+        } else {
+          buddyIndex = startingIndex + 2;
+        }
+        break;
+      default:
+        return;
+    }
+    if (appendInstead) {
+      sortItem.parentElement.appendChild(sortItem);
+    } else {
+      sortItem.parentElement.insertBefore(sortItem, this.items[buddyIndex]);
+    }
+
+    this.items = Array.from(this.el.children);
+
+    event.detail.handle.activated = true;
+    event.detail.handle.shadowRoot.querySelector(this.handleSelector).focus();
```

Comment:
fixme: I think we should add a `setFocus()` method on the handle instead of going through the shadowRoot. We already do this for the `calcite-action` and `calcite-panel`

Reply:
@pr3tori4n this wouldn't work in Edge anyway

Reply:
It should now that Edge is on Chromium shouldn't it?

Reply:
> It should now that Edge is on Chromium shouldn't it?

We should support older edge users regardless.